admin/en.json

```diff
@@ -188,7 +188,7 @@
     "TELEMETRY_SUCCESS": "Telemetry Configuration was saved successfully",
     "TELEMETRY_ERROR_STATUS": "Error",
     "TELEMETRY_ERROR_MESSAGE": "Your configuration could not be saved. Please try again.",
-    "TELEMETRY_FETCH_DATA_ERROR_MESSAGE": "Telemetry data fetching failed:",
+    "DIAGNOSTICS_FETCH_DATA_ERROR_MESSAGE": "Telemetry data fetching failed: {{msg}}",
```

Comment:
I changed the name of the token from `TELEMETRY_FETCH_DATA_ERROR_MESSAGE` to `DIAGNOSTICS_FETCH_DATA_ERROR_MESSAGE`. Also, I added a `msg` placeholder.